packages/terra-icon/tests/wdio/icon-spec.js

```diff
@@ -1,4 +1,4 @@
-Terra.describeViewports('Icon', ['large'], () => {
+Terra.describeViewports('Icon', ['huge'], () => {
```

Comment:
By changing this viewport, all icon screenshots need to be deleted and re-generated

Reply:
That should be reflected in [b30e0f1](https://github.com/cerner/terra-core/pull/2783/commits/b30e0f19076be91ba18eeb6633d8c2c64db39bc3).